xml2rfc_compat/serializers/__init__.py

```diff
@@ -64,6 +64,14 @@ def serialize(
     if anchor:
         root.set('anchor', anchor)
 
+    # Fill in appropriate target
```

Comment:
Without running this code, shouldn't this only happen when `is_referencegroup` is `true`?

Reply:
Yes, that's correct. I'll fix it